packages/server/graphql/types/JiraServerIntegration.ts

```diff
@@ -136,7 +135,18 @@ const JiraServerIntegration = new GraphQLObjectType<{teamId: string; userId: str
           (projectKeyFilter) => IntegrationRepoId.split(projectKeyFilter).projectKey!
         )
 
-        const issueRes = await integrationManager.getIssues(queryString, isJQL, projectKeys)
+        // "first + 1" is a trick to make pagination works easier, see hasNextPage
+        const maxResults = first + 1
+        // Cursor must be a string
```

Comment:
+1 It's always better to state why something is the case, i.e. why does the cursor need to be a string?